data/projects.json

```
@@ -138,6 +138,16 @@
         "level": "basic",
         "site": "",
         "tags": "car-game,c,c++,graphics-project"
+    },
+    {
+        "name": "Tamagotchi-Game",
+        "display": "Tamagotchi Game",
+        "description": "A game where you manage a creature with 3 needs (usually eat, sleep, entertainment) and as your creature ages, it morphs.",
+        "author": "NikkiHmltn",
+        "category": "game",
+        "level": "beginner",
```

The level only support basic, intermediate, and advance
```suggestion
        "level": "basic",
```

---

I made the requested changes to the idea. Thank you for the feedback!